docs/installation.rst

```diff
@@ -70,3 +70,36 @@ Once your configuration is done, you can launch the deployment :
 .. code-block:: bash
 
     (venv) $ enos deploy
+
+
+The deployment is the combination of the following three phases:
+
+1. Acquire raw resources that are necessary for the deployment of
+   OpenStack. Enos acquires resources according to the ``provider``
+   and ``resources`` information in the reservation file. Someone can
+   perform this phase by calling ``enos up``.
+
+2. Deploy OpenStack on resources gotten during the previous phase.
```

Comment:
"Deploy OpenStack to the resources acquired during"

Reply:
Done